### DISCUSSION TITLE
Rename `wasm-pkg-loader` -> `wasm-pkg-client`

### DISCUSSION THREAD

**Discussion by lann** [bytecodealliance/wasm-pkg-tools#54] [General]
As I've been looking at unified publishing (#13) it has become clear to me that my plan to have separate `wasm-pkg-loader` and `wasm-pkg-publisher` crates is arbitrary and unhelpful; the two crates would have ended up with nearly identical dependencies and large amounts of code overlap. If there end up being scenarios where a downstream consumer wants to avoid some code associated exclusively with publishing then we can feature flag it out instead.

Given that, I'm planning to rename `wasm-pkg-loader` to `wasm-pkg-client` as part of the publishing implementation. I'd like to just drop the `wasm-pkg-loader` crate entirely but if that is going to cause pain for someone already depending on it please comment here and we can look at a more gradual migration.

**Comment by calvinrp** [comment]
Go for it

**Comment by thomastaylor312** [comment]
Feel free to combine! I'm currently using loader in my work with cargo component, but as long as things mostly work the same, I can easily adapt
